web/modules/custom/error_page/src/EventSubscriber/ErrorPageFinalExceptionSubscriber.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\error_page\EventSubscriber;
+
+use Drupal\error_page\ErrorPageRenderer;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Subscribes to KernelEvents::EXCEPTION events.
+ */
+class ErrorPageFinalExceptionSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      // This subscriber steals the show by acting just before the core
+      // FinalExceptionSubscriber, which has the priority set to -256.
+      // @see \Drupal\Core\EventSubscriber\FinalExceptionSubscriber::getSubscribedEvents()
+      KernelEvents::EXCEPTION => ['onException', -255],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function onException(GetResponseForExceptionEvent $event) {
+    $uuid = !empty($event->uuid) ? $event->uuid : NULL;
```

Comment:
Compared to the `\Drupal\Core\EventSubscriber\FinalExceptionSubscriber`, this code completely discards the current website configuration regarding the error level reporting. We should instead honour the user configuration.
As written above in the documentation remark, this module should add identifiers to allow easier reporting from users and easier debugging of production errors but not completely hide errors on uat server for example.